modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/impl/RocksDbSpillout.java

```diff
@@ -144,6 +144,7 @@ public RocksDbSpillout(
         this.groupEndBound = new Slice(groupEndPrefix);
 
         this.writeOptions = new WriteOptions();
+        this.writeOptions.setDisableWAL(true);
```

Comment:
How will we maintain consistency in saving commands?

Reply:
It is related to volatile storage. When starting a node, we recreate this RocksDB effectively destroying its state.

Reply:
It was simply not obvious to me from the name of the class that it was for volatile tables and it was said in the documentation at the very end.
Maybe I would rename the class to something like `RocksDbLogsNotSpillout`. 
Or at least indicate in the title of the problem that it is about volatile raft groups.

Reply:
Included 'volatile' in the title of the Jira issue and PR